### PR TITLE
clean up utils, add some comments

### PR DESCRIPTION
**Pull request by andre-merzky**
clean up utils, add some comments

### CODE REVIEW

File: src/radical/entk/utils/sync_initiator.py [hunk 1]
```diff
@@ -39,25 +49,40 @@ def sync_with_master(obj, obj_type, channel, queue, logger, local_prof):
...
+
+      # self._logger.info('waiting for ack')
 
         method_frame, props, body = channel.basic_get(queue=reply_queue)
 
         if body:
+
+            # AM: what happens to messages which don't match the `if` below
+            # - it seems like they are doscarded?  Why?
+
```
Comment (vivek-bala) [inline src/radical/entk/utils/sync_initiator.py]: Messages from RMQ can come out of order. Messages that are unacknowledged will
get resent, so no messages are discarded actually. We acknowledge a processed
message in line 82.

Thanks for noticing that. I will an inline comment regarding this.

File: src/radical/entk/utils/init_transition.py [hunk 2]
```diff
@@ -28,9 +28,14 @@ def transition(obj, obj_type, new_state, channel, queue, profiler, logger):
...
-    except Exception, ex:
 
-        logger.exception('Transition of %s to %s state failed, error: %s' % (obj.uid, new_state, ex))
+    except Exception:
+
+        logger.exception('Transition of %s to %s state failed,',
+                         obj.uid, new_state)
+
+        # AM: why the reset if the transition failed?  Or is that
+        # non-transactional after all?
```
Comment (vivek-bala) [inline src/radical/entk/utils/init_transition.py]: The try-except block together make the transition function transactional. 
sync_with_master() is not transactional. 

Does that help? Maybe I am missing something in the question.
Let me know.

Comment (andre-merzky) [reply src/radical/entk/utils/init_transition.py]: @vivek-bala ping?

File: src/radical/entk/utils/sync_initiator.py (hunk 1 above)
Comment (andre-merzky) [reply src/radical/entk/utils/sync_initiator.py]: @vivek-bala ping?

Comment (andre-merzky) [reply src/radical/entk/utils/sync_initiator.py]: @vivek-bala: ping?

File: src/radical/entk/utils/init_transition.py (hunk 2 above)
Comment (andre-merzky) [reply src/radical/entk/utils/init_transition.py]: @vivek-bala: ping?

**Comment by andre-merzky** [comment]
Another one bites the dust - this does not merge anymore... :-/

**Pull request closed without merging**